cli/utils.js

```diff
@@ -99,7 +99,7 @@ const initHelp = (program, cliName) => {
             logger.info(`Did you mean ${chalk.yellow(suggestion)}?`);
         }
 
-        process.exit(1);
+        process.exitCode = 1;
```

Comment:
Probably we should throw an exception here too.
This seems a 'command not found'.

Reply:
Again I've just tested this function and even when we set `process.exitCode=1` it seems to exit cleanly without any problems. :smile:

Reply:
Only place is called is at initHelp:
https://github.com/jhipster/generator-jhipster/blob/08f36bbe77a92140516dc4a50401a202c9001f38/cli/cli.js#L87-L101
Letting the process continue seems to make no harm.